react-blocks/src/styles/blocks/Happypoint.scss

```diff
@@ -0,0 +1,9 @@
+#editor .editor-writing-flow > div > div > div > .editor-block-list__layout > div[data-type="planet4-blocks/happypoint"] {
```

Comment:
please remove these to keep blocks layout consistent on the editor.
the second rule is not used.